README.md

```diff
@@ -106,12 +106,30 @@ follow_links: True # Boolean
 ignore_names_file: .*/test # regex
 fail_under: 90 # int 
 percentage_only: True # Boolean
-
+ignore_patterns:
```

Comment:
Can we add something like ` # Dict with key/value pairs of file-pattern/node-pattern` to match the other options? There's probably a better way to word it...